src/tools/rbd_cache/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_executable(rbd-cache
```

Comment:
Note: the cache controller really shouldn't be dependent upon RBD -- it's just a RADOS persistent, read-only object cache. Therefore, it could be something like ```ceph-immutable-object-cache``` (or similar). It should just require pool id, oid, and byte ranges.

Reply:
I see, will change the API and make this daemon to work at RADOS level